.coveragerc

```diff
@@ -244,6 +244,8 @@ omit =
     homeassistant/components/econet/const.py
     homeassistant/components/econet/sensor.py
     homeassistant/components/econet/water_heater.py
+    homeassistant/components/ecowitt/__init__.py
+    homeassistant/components/ecowitt/sensor.py
```

Comment:
While the review is in progress, why not add coverage...

Reply:
I'm working on it little by little, but, these two are hard.

Reply:
I need to rethink how I do the config_flow tests here..  this will take a little while..